src/TwillRoutes.php

```diff
@@ -402,4 +402,18 @@ public function getAuthRedirectPath(): string
         return config('twill.auth_login_redirect_path')
             ?? rtrim(config('twill.admin_app_url') ?: '', '/') . '/' . ltrim(config('twill.admin_app_path') ?? 'admin', '/');
     }
+
+    public function isTwillRequest(): bool
```

Comment:
I think we have something like this elsewhere, will look for it.

Reply:
I did a search on 'twill.admin_app_url' and 'twill.admin_app_path' and found nothing resembling this (except non exposed in the ExceptionHandler which I just removed)

Reply:
You're right, I was remembering this from a packaged capsule, not in the core.